### ISSUE TITLE
model.pt, val_loss_log.npy 제출 방식 문의

### ISSUE THREAD

**Issue by hansmaan** [LISTatSNU/FastMRI_challenge#350]
안녕하세요. Room 218 팀의 한창희입니다.

Final Submission의 경우, model.pt, val_loss_log.npy 제출이 요구되고, Mid Submission에서는 val_loss_log.npy만 요구되는 것으로 이해했습니다. (추가 GPU 원할 경우)

**이 제출 요구 사항에 대해서 3가지가 궁금합니다.**

저희가 MoE 구조를 구현하려 하고 있고, 서로 다른 모델을 각각 학습시켜서, 마지막에 합친 후, Reconstruct하는 것을 생각하고 있었습니다.
이렇게 할 경우, **model.pt와 val_loss_log.npy를 각 모델에 대해 1개씩**, 여러 개 올려도 괜찮을지가 궁금합니다.

또한, 저희가 Knee와 Brain인지 구별만 하는 간단한 Neural Network를 구현했는데, **이 Net은 Loss가 SSIM일 수 없습니다.** 저희는 그냥 정확도를 Loss 삼아서 보는데, 이 Network의 **log에는 SSIM이 아닌 Accuracy가 적혀 있어도 괜찮은지** 궁금합니다.

마지막으로, 한 번의 Train Session으로 진행되어야 하는지 궁금합니다. 저희 같은 경우, 30 epoch 1번 돌리고, 거기에 또 40 epoch 돌리고, 이런 식으로 추가 실험을 얹고 있습니다. 이렇게 하게 되면, 로그 파일이 여러 개 나올 것 같은데, 이렇게 되어도 괜찮은지 궁금합니다!

참고로, 스크립트 하나만 실행해도 전 과정을 안정적으로 수행할 수 있도록, 스크립트를 구현할 예정입니다.

감사합니다!

**Comment by hansmaan** [comment]
1번 질문 관련하여, 보충 설명을 드리자면
이전 우승자 분들 중에, Parameter Freezing 형태로 각 모델을 독립적으로 학습시킨 것을 보았는데 그런 식으로 해야 될지,
각 모델별 학습 각각에 대해서, val_loss_log.npy를 드려도 될지?

**Comment by Racul** [comment]
@hansmaan 
조교 박정민입니다.

1. 모델별로 .pt, .npy를 올려도 됩니다.
2. loss가 ssim일 필요는 없습니다. 재현 가능하면 됩니다.
3. 하나의 모델을 여러번 실행해도 됩니다. (단, 조교가 단순히 스크립트를 차례대로 구동하는 것으로 작동해야 합니다)
